packages/frontend/web-ui/src/game/hooks/useCreateNewGame.spec.ts

```diff
@@ -0,0 +1,399 @@
+jest.mock('../helpers/validateNumberOfPlayers');
+jest.mock('../../common/helpers/joinGame');
+jest.mock('../../common/helpers/getUserMeId');
+jest.mock('../../common/http/services/HttpService');
```

Comment:
why are we mocking the `httpClient`? It shouldn't be used. `useCreateGame`should be mocked instead

Reply:
Done! I've changed the test's body. Also, I've added an extra validation in the second useEffect on useCreateNewGame for validate the response result with the status is pending Backend.